docs/competition/matches/match-scoring.md

```diff
@@ -11,47 +11,76 @@ Any roles related to scoring matches go here.
 
 If you have any questions you may first ask the head match scorer and secondly the judge.
 
 ### Template Arena Setup Checklist
 
-1. Tokens in the arena aren't damaged. If they are, swap them out with the spares.
+1. Props (e.g: tokens) in the arena aren't damaged. If they are, swap them out with the spares.
 2. The tokens in the arena are intended for that arena -- this can be established by reading the grey text printed on each marker on the token.
    If there is more than one arena then the markers on the sides of any tokens will have the name of the arena as part of the text in the bottom left of the marker.
 3. There must be the correct number of tokens of each type in the arena. The precise configuration depends on the game, though the per-year version of this checklist should be explicit about what's needed (and not rely on volunteers remembering the rules)
 4. Each token is placed on a suitable token starting positions that are marked out on the arena floor. Where possible these should be colour-coordinated.
 
-![Arena Diagram](https://cloud.githubusercontent.com/assets/510498/14783612/a9026b9a-0ae8-11e6-89ef-678acd9ec48a.png)
+![Arena Diagram](diagrams/arena.svg)
 
 ### Robot Damage Checklist
 
 - If the damage is intentional, seek the judge.
 - If the damage is unintentional:
   - If a robot mashes into another robot, but only briefly, leaving both robots free to move (including situations in which, say, one robot has a fragile, flaily arm that gets snapped off in the collision) then you must leave the robots to continue -- the teams are aware that accidental bumps and scrapes are inevitable.
-  - If a robot mashes into another robot, and pins it, stopping it from being able to function (including one dragging the other around if they become intertwined) then at least one of the robots must be switched off, depending on which robot caused the damage, but the scores are **not** forfeited in this case.
+  - If a robot mashes into another robot, and pins it, stopping it from being able to function (including one dragging the other around if they become strongly intertwined) then at least one of the robots must be switched off, depending on which robot caused the damage, but the scores are **not** forfeited in this case.
 
 ## Match Score Entry
 
-**Responsible for receiving scores from the match scorers and entering them into the competition software. They may not bring in any subordinate volunteers.**
+**Responsible for receiving match states from the marshals and entering them into the competition software. They may not bring in any subordinate volunteers.**
 
 In the role of score entry you will be stationed at a desk near to the arena/staging area. When a score sheet is given to you by the match scorers, you will enter the scores into a computer and then file the score sheet in a folder.
```

Comment:
Maybe also clarify that this is a two-person role, with the idea being that one person enters the scores while another checks that they're entered correctly?

We could also mention that they'll be given _pairs_ of sheets and that they should check that the pair agree as a first step.

Reply:
1edb662fb2de00ea1f03a68f97bd37b05957b4dbfixed in